packages/editor-ui/src/components/NodeExecuteButton.vue

```diff
@@ -229,6 +247,24 @@ export default defineComponent({
 					});
 					this.$emit('execute');
 				}
+
+				if (this.nodeType && this.nodeType.name === FORM_TRIGGER_NODE_TYPE) {
+					const testUrl = this.getWebhookUrl(this.nodeType.webhooks[0], this.node, 'test');
+
+					const windowWidth = window.innerWidth;
+					const smallScreen = windowWidth <= 800;
+					if (smallScreen) {
+						window.open(testUrl, '_blank');
+					} else {
+						const width = 700;
+						const height = window.innerHeight - 50;
+						const left = (window.innerWidth - width) / 2;
+						const top = 50;
+						const features = `width=${width},height=${height},left=${left},top=${top},resizable=yes,scrollbars=yes`;
+
+						window.open(testUrl, '_blank', features);
+					}
```

Comment:
maybe this can be a helper function? need to reuse that for other execute buttons in the canvas too?
https://www.notion.so/n8n/n8n-Form-Trigger-385708f93d8b4c698cb7573b4ff8b997?pvs=4#da2df29b29414b8abb1e18b1de81528f